go.mod

```diff
@@ -0,0 +1,149 @@
+	github.com/matttproud/golang_protobuf_extensions v1.0.1
+	github.com/miekg/dns v1.1.0
+	github.com/mitchellh/go-homedir v1.0.0
+	github.com/mitchellh/mapstructure v1.0.0
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742
+	github.com/onsi/ginkgo v0.0.0-20180312103414-ba8e856bb854
+	github.com/onsi/gomega v0.0.0-20171012021143-283ed655c6b8
+	github.com/op/go-logging v0.0.0-20160206111733-0882c9abce53
+	github.com/opencontainers/go-digest v1.0.0-rc1
+	github.com/opencontainers/image-spec v1.0.1
+	github.com/opencontainers/runc v0.1.1
+	github.com/opencontainers/runtime-spec v1.0.1
+	github.com/pborman/uuid v0.0.0-20160208165331-14801136da12
+	github.com/pelletier/go-buffruneio v0.2.0
+	github.com/pelletier/go-toml v1.2.0
+	github.com/petar/GoLLRB v0.0.0-20130427215148-53be0d36a84c
+	github.com/peterbourgon/diskv v2.0.1+incompatible
+	github.com/petermattis/goid v0.0.0-20180202154549-b0b1615b78e5
+	github.com/pkg/errors v0.8.0
+	github.com/pmezard/go-difflib v1.0.0
+	github.com/prometheus/client_golang v0.9.0-pre1
+	github.com/prometheus/client_model v0.0.0-20180712105110-5c3871d89910
+	github.com/prometheus/common v0.0.0-20180801064454-c7de2306084e
+	github.com/prometheus/procfs v0.0.0-20180920065004-418d78d0b9a7
+	github.com/russross/blackfriday v0.0.0-20171011182219-6d1ef893fcb0
+	github.com/sasha-s/go-deadlock v0.0.0-20180822072021-d68e2bc52ae3
+	github.com/servak/go-fastping v0.0.0-20160802140958-5718d12e20a0
+	github.com/shirou/gopsutil v2.18.10+incompatible
+	github.com/shirou/w32 v0.0.0-20160930032740-bb4de0191aa4
+	github.com/sirupsen/logrus v1.0.3
+	github.com/spf13/afero v1.1.2
+	github.com/spf13/cast v1.2.0
+	github.com/spf13/cobra v0.0.0-20181127133106-d2d81d9a96e2
+	github.com/spf13/jwalterweatherman v1.0.0
+	github.com/spf13/pflag v0.0.0-20181223182923-24fa6976df40
+	github.com/spf13/viper v1.3.1
+	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2
+	github.com/tylerb/graceful v0.0.0-20170103162203-d37e108c8976
+	github.com/vishvananda/netlink v0.0.0-20181018205019-d3a23fd178f1
+	github.com/vishvananda/netns v0.0.0-20180720170159-13995c7128cc
+	golang.org/x/crypto v0.0.0-20180531191117-5ba7f6308246
+	golang.org/x/net v0.0.0-20170922011244-0744d001aa84
+	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
+	golang.org/x/sync v0.0.0-20181108010431-42b317875d0f
+	golang.org/x/sys v0.0.0-20181122145206-62eef0e2fa9b
+	golang.org/x/text v0.3.0
+	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2
+	golang.org/x/tools v0.0.0-20180928123317-84988e2dba31
+	google.golang.org/appengine v1.2.0
+	google.golang.org/genproto v0.0.0-20170904050139-595979c8a7bf
+	google.golang.org/grpc v1.11.3
+	gopkg.in/check.v1 v1.0.0-20160105164936-4f90aeace3a2
+	gopkg.in/fsnotify.v1 v1.4.7
+	gopkg.in/inf.v0 v0.9.1
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3
+	gopkg.in/yaml.v2 v2.2.1
+	k8s.io/api v0.0.0-20190111032252-67edc246be36
```

Comment:
Versions of k8s libraries still look weird. I remeber some discussions in k8s community about prividing go.mod-compatible versioning. Maybe you just need to update all k8s deps to the newest versions or commits?